resources/views/site/landingPageDetail.blade.php

```diff
@@ -4,12 +4,12 @@
 
     @include('site.landingPage._header----'.$landingPageType)
 
-    <div class="o-landingpage__body o-blocks">
+    <div class="o-landingpage__body {{ $landingPageType === 'pre-made' ? 'aic-ct-landingpage__body' : '' }} o-blocks">
```

Comment:
I'm wondering if `aic-ct-landingpage__body` should be `o-landingpage__body--aic-ct`, as it is _technically_ a modifier

Reply:
Good point, I've updated.